numpy/array_api/_array_object.py

```diff
@@ -56,7 +56,7 @@ class Array:
     functions, such as asarray().
 
     """
-    _array: np.ndarray
+    _array: np.ndarray[Any, Any]
```

Comment:
(This would be an exception, since this file is actually typed)

Reply:
Ooops?  This is gone, but the `dict[Any, Any]` above in the python file is still there?